templates/year/problems/problemB-P.tex.vyfuk

```diff
@@ -23,10 +23,10 @@
 \probsolution[cs]{%
 }
 
-% TODO text zadání
-% TODO odborné korektury zadání
-% TODO jazykové korektury zadání
-% TODO typo korektury zadání
+% text zadání
+% odborné korektury zadání
+% jazykové korektury zadání
+% typo korektury zadání
```

Comment:
Ty značky `TODO` se kdysi používaly v "aplikaci" na sledování workflow úlohy. Myslím, že už to ale není v provozu. Pokud není jiný důvod pro tyhle komentáře, tak můžou být odstraněny zcela.

Reply:
Výfuk to stále používá, jak z hlediska přehledu, co je hotovo, tak z hlediska pro hledání scriptem. Jediné co, tak to nebylo jednotné s využitím `TODO`, proto jsem to sjednotil.